ui/packages/app/src/api_docs/ResponseObject.svelte

```diff
@@ -26,7 +26,7 @@
 		&nbsp;&nbsp;"data": [
 		<br />
 		{#each dependency.outputs as component_id, component_index}
-			&nbsp;&nbsp;&nbsp;{#if dependency_outputs[dependency_index][component_index] !== undefined}
+			&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{#if dependency_outputs[dependency_index][component_index] !== undefined}
```

Comment:
Probably a better way to do this? Idk lol

Reply:
Can't we just apply `whitespace: pre` or something?

Reply:
I did this with margins 

![Screen Shot 2023-02-22 at 6 53 10 PM](https://user-images.githubusercontent.com/9021060/220811463-b0288162-931a-47b7-bee1-9e8ab36502af.png)